### ISSUE TITLE
OCR mistakes are unintuitive to correct before initial save

### ISSUE THREAD

**Issue by zuzak** [edent/openbenches#195]
I just uploaded some benches for the first time.
The user flow was really good, except when there were some problems with the OCR, which I didn't notice until I'd already saved, meaning that the [@openbenches](https://twitter.com/openbenches/status/1133726069729714179) twitter account tweeted some bogus inscriptions.

This happened because the textarea to edit the inscription is delayed slightly.
In that time, I uploaded more photos of the bench, meaning that when the inscription box appeared, it was already scrolled off the screen. As I didn't know the inscription box was a thing, I missed it entirely and saved.

It would be good to make it more obvious. Maybe it should force a photo with a close-up inscription to be at the bottom, so that it's near the save button. Or perhaps it could prompt you to check the inscription before saving?

**Comment by edent** [comment]
Good point. I'll look into it.

**Comment by arizonagroovejet** [comment]
I read this issue and thought "but it tells you that it's doing text detection so you know to wait for that to complete and check it". But I've just tried selecting photos on  `/add/` several times (reload page between attempts) and I'm not seeing any such message. I was so sure I'd seen such a message I went looking for it in the code. 

There is a message that says text detection is happening:
https://github.com/edent/openbenches/blob/b81e8bf838d0b9b5cf0d94fe30fe6768c3e9e008/www/libs/vision/vision.js#L94
But it's a child of an element that has `display:none` set 
https://github.com/edent/openbenches/blob/0f316fe778a1bd5f2c0d0691aa3e7269fd35f282/www/add.php#L41
until automatic text detection has completed
https://github.com/edent/openbenches/blob/b81e8bf838d0b9b5cf0d94fe30fe6768c3e9e008/www/libs/vision/vision.js#L102

I suspect the message used to be visible whilst text detection was happening but that changed at some point and I've added enough benches that I've pretty much stopped paying attention to what the `/add/` page says because I already know how it works.
